.github/workflows/python-app.yml

```diff
@@ -42,3 +42,16 @@ jobs:
     - name: Test with python unittest
       run: |
         python -m unittest -v -b
+    - name: Distribute to test pypi
+      if: ${{ github.event_name == 'pull_request' }}
+      run: |
+        TEST_PYPI="True"
+        set | grep TEST_PYPI
+        export TEST_PYPI
+        python -m build
+        python -m twine upload --repository-url https://test.pypi.org/legacy/ -u __token__ -p ${{ secrets.AEROSPIKE_TEST_PYPI_TOKEN }} dist/* --verbose
```

Comment:
@MicahCarrick Does this usage of secrets look correct to you? you mentioned called and `environment` of `pypi-env`, but I am not sure what this usage looks like.

Reply:
The secrets must be part of an "environment" which then can be referenced in the workflow: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idenvironment

Reply:
And: https://docs.github.com/en/actions/reference/environments